### DISCUSSION TITLE
Why doesn't <Await/> offer a `fallback` prop and handle <Suspend/> wrapping?

### DISCUSSION THREAD

**Discussion by maranomynet** [remix-run/remix#5582] [Q&A] [answered]
If there is a reason for this, it might be worth mentioning in the docs.

**Comment by kiliman** [comment, marked as answer]
Remix prefers to expose building blocks so devs can use them in whatever way makes sense. In this case, allowing control over your own `<Suspense>` boundaries. For example, having multiple `<Await>` in a single parent `<Suspense>`

You could always create your own `<AwaitSuspense>` wrapper.

**Reply by maranomynet** [reply]
Thanks. That's quite reasonable.

Might be worth a mention in the docs, though.
